### PR TITLE
Aric/phptest

### PR DESCRIPTION
**Pull request by AricStewart**

### CODE REVIEW

File: api/src/Tests/Traits/AppTestTrait.php [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    }
+
+
+    /* End */
+}
+
+/**
+ * App Test Trait.
+ */
+trait AppTestTrait
```
Comment (mshappe) [inline api/src/Tests/Traits/AppTestTrait.php:44]: TIL!

(Rails would call this a Module, same basic behaviour. Beautiful).

**Comment by mshappe** [comment]
This is excellent! I particularly like your leveraging of PHP `trait`s, which I didn't know about until today, to DRY up the baseline setup of tests!

I am looking forward to wiring up a CI/CD pipeline to run these!

File: api/src/Tests/Traits/AppTestTrait.php (hunk 1 above)
Comment (AricStewart) [reply api/src/Tests/Traits/AppTestTrait.php:44]: yay!

**Comment by AricStewart** [comment]
Agreed. I am trying to hook the SonarCloud report to it but it seems to be unable to find the coverage report. I will keep poking.

**Review by mshappe — DISMISSED** [review]

**Pull request merged**